src/Application.cpp

```diff
@@ -52,6 +52,8 @@ along with RALibretro.  If not, see <http://www.gnu.org/licenses/>.
 
 #define TAG "[APP] "
 
+#include <thread>
```

Comment:
Why is this necessary?

Reply:
it isn't (anymore)

✔️ https://github.com/theMK2k/RALibretro/commit/86769071666acd9383bcfb512bb414e374548a10